### DISCUSSION TITLE
osu!mania Compactible Buttons

### DISCUSSION THREAD

**Discussion by CamelliaNotReal** [ppy/osu#16867] [Ideas] [answered]
Some players on mobile want to play osu!mania, but there is no buttons to add it. So i want you to add some osu!mania buttons.

osu!mania buttons can be placed in the Center-bottom of the screen and the buttons can be customise by dragging in its positions also.

If you wish to add it, i will continue to support your game as well.

**Comment by peppy** [comment, marked as answer]
Please search first. This has been brought up about 300 times.
